### PR TITLE
chore(android): add null checks

### PR DESCRIPTION
**Pull request by KrzysztofMoch**
## Summary
Add some null checks 

### Motivation
Fix edge case crashes

## Test plan
- should not crash app now

### CODE REVIEW

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -900,25 +900,33 @@ public void onServiceConnected(ComponentName name, IBinder service) {
...
+                    }
                 } catch (Exception ignored) {}
 
                 playbackServiceBinder = null;
             }
 
             @Override
             public void onNullBinding(ComponentName name) {
-                DebugLog.e(TAG, "Cloud not register ExoPlayer");
+                DebugLog.e(TAG, "Could not register ExoPlayer");
```
Comment (freeboub) [inline android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:929]: : ❤️

File: android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 2]
```diff
@@ -900,25 +900,33 @@ public void onServiceConnected(ComponentName name, IBinder service) {
                 playbackServiceBinder = (PlaybackServiceBinder) service;
 
                 try {
-                    playbackServiceBinder.getService().registerPlayer(player,
-                            Objects.requireNonNull((Class<Activity>) (themedReactContext.getCurrentActivity()).getClass()));
+                    Activity currentActivity = themedReactContext.getCurrentActivity();
```
Comment (freeboub) [inline android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:903]: I found such issue when starting the app while screen is locked ... Not sure if it helps ...

**Review by freeboub — APPROVED** [review]
